### ISSUE TITLE
import: Interface confusing

### ISSUE THREAD

**Issue by sophie-h** [bottlesdevs/Bottles#2470]
![image](https://user-images.githubusercontent.com/3466497/210150593-6b310260-c8ab-4efb-b713-45688ce755b5.png)

I already mentioned this one 8 months ago. Now, 'Importer' got replaced with some technical terms.

How am I supposed to know what a prefix is? Can't Bottles check if it has access to them? Also, the import icon does not look very intuitive to me.

I wonder if the bottles imports can't be in the main menu on the main view or do the "Prefixes" view play any role in importing bottles?

**Comment by nPHYN1T3** [comment]
I'll 1up you for this because my question is why the hell is it looking for prefixes at all? When I go to "import" I'm expecting to "import" an exported wine config. So if you have a bottle and use the export configuration (yml file) how the hell do you import it? There is nothing in the docs about this and the fact this looks for prefixes rather than yml files is whack. So is this for Importing full backups or what? If so what purpose is the config export when there is seemingly no way to apply it to a new bottle prefix? Why don't we have quantum socks yet so when one goes missing in the dryer the entanglement will allow a tunnel to find it? (Just seeing if you were paying attention.)

**Comment by TheEvilSkeleton** [comment]
@sophie-h I agree that the UI is extremely confusing. To add more, "full archive" doesn't really mean much:
![image](https://user-images.githubusercontent.com/50847364/210637939-9853bdeb-baa9-42c2-8af8-98ea893c3676.png)
i.e. are we talking about backups? _Any_ tar.gz archive? etc.

**Comment by nPHYN1T3** [comment]
Kinda classic security issue where you assume and trust the correct info will be given. While I originally found this post because I was trying to find how to apply yml backups I've been using Bottles a lot lately after a hiatus and the UI is just disjointed. Opens new windows pointlessly, doesn't give you the option to delete say a runner after an install unless you close and reopen. Often times doesn't show what is installed or showed installed things as not installed. It's confusing by design but then it's also broken making the confusion of dealing with things worse.

**Comment by TheEvilSkeleton** [comment]
@nPHYN1T3 Please, don't leave comments if you are just going to rant about the UI.

**Comment by nPHYN1T3** [comment]
You can call it ranting but saying "it's confusing" is about the same as "it doesn't work." Hard to fix "vagueness." My rants detail some key issues that contribute to it being confusing and broken. Certainly not an exhaustive list but as a developer I'd rather some detail than "it doesn't work."

**Comment by TheEvilSkeleton** [comment]
> I wonder if the bottles imports can't be in the main menu on the main view or do the "Prefixes" view play any role in importing bottles?

Is this what you meant?

![image](https://user-images.githubusercontent.com/50847364/210647916-8db85a59-91d4-4f89-a2e8-ae3f7b815c2d.png)

**Comment by TheEvilSkeleton** [comment]
@sophie-h is this good?

![image](https://user-images.githubusercontent.com/50847364/215304636-07ad1df5-d56e-4505-9167-29fe6e42dd06.png)

**Comment by bckelley** [comment]
Confusing the damn thing doesn't work!
